astropy/coordinates/tests/test_api_ape5.py

```diff
@@ -12,17 +12,17 @@
 deviations from the original APE5 plan.
 """
 
+import pytest
+
 import numpy as np
 from numpy.random import randn
 from numpy import testing as npt
-from ...tests.helper import (pytest, quantity_allclose as allclose,
+from ...tests.helper import (quantity_allclose as allclose,
                              assert_quantity_allclose as assert_allclose)
-raises = pytest.raises
 
 from ... import units as u
 from ... import time
 from ... import coordinates as coords
-from ..errors import *
```

Comment:
Either that's a good catch or something very weird. But the tests passed ...

Reply:
None of those errors/warnings got used in this file, thus the removal.

Reply:
In that case: Good catch 👍